### ISSUE TITLE
Record on sensor

### ISSUE THREAD

**Issue by lucasliebe** [Sensors-in-Paradise/android-native-xsens#36]
## Description

Recording on Sensors will hopefully give better data

## Must Include

- [x] Record and save data on sensor
- [x] Test if recording + data transfer is possible and how long transfer takes via Bluetooth
- [ ] Transfer to App on Stop Recording see #57 

## Notes

This is an issue to test this out so please try to see if it is practical or brings improvements, present your information the next meeting, don't merge into master before.

**Comment by treyfel** [comment]
Which data quantities do we want to export? I could simply export the same we currently use when logging, but I think we need some consensus here in the future(data ecploration).

**Comment by treyfel** [comment]
What does the second Task mean? Do we immediately want to export data onto the connected phone? I think this would destroy the purpose of saving on the devices, since we would have to make sure, that the devices are properly connected when stopping the recording. IMO: we should make exporting the data a separate functionality... sth  like a button: exportAll,  that exports all the recorded data to a connected devices, ensuring proper connection and deleting the exported data on all sensors afterwards.

**Comment by lucasliebe** [comment]
First of all I think it would be great to if possible export as much data as possible right now, then the ML Team can modify the data and try out what works best for activity recognition

Second the second task was meant to export in the background after finishing a recording. The sensors would need to be connected to properly stop the recording anyway, but you are right that somehow we need to preserve Data Loss. Maybe some other people can also give their opinion on this. For instance @tfiedlerdev, @kpostnov or @FranzSw if you have any thoughts on how we should do this please note it here. If the recording time on the new sensors is multiple hours anyway, maybe we can record a full day and then just transfer data back? Tho we wouldnt get as much feedback on our recordings like now.

**Comment by treyfel** [comment]
The thing with exporting the data while recording is that it takes quite a bit of time (10 min exporting for 60 min recording wih 60Hz). This would mean a break from recording, everytime we want to export?

**Comment by treyfel** [comment]
Still we'd need to ensure a stable connection during the export!

**Comment by lucasliebe** [comment]
we discussed today that we want to have the export after we finished the recording. if files are transferred completely the app will signalize to the sensors so they can delete the recording. starting a new recording while exporting needs to be possible for this!

**Comment by treyfel** [comment]
hab kb mehr lol

**Comment by lucasliebe** [comment]
this functionality is deprecated and will not be implemented in our scope
